web/src/components/tasks/business-formation/review/ReviewStep.tsx

```diff
@@ -32,62 +34,73 @@ export const ReviewStep = (): ReactElement => {
-          </ReviewSection>
-        )}
-        {isLP && (
-          <ReviewSection
-            buttonText={Config.formation.general.editButtonText}
-            header={Config.formation.fields.withdrawals.label}
-            stepName={"Business"}
-            testId="edit-withdrawls-step"
-          >
-            <ReviewText fieldName={"withdrawals"} />
-          </ReviewSection>
-        )}
-        {isLP && <ReviewPartnership />}
-        {isLP && (
-          <ReviewSection
-            buttonText={Config.formation.general.editButtonText}
-            header={Config.formation.fields.dissolution.label}
-            stepName={"Business"}
-            testId="edit-dissolution-step"
-          >
-            <ReviewText fieldName={"dissolution"} />
-          </ReviewSection>
-        )}
-        {hasProvisions && <ReviewProvisions />}
-        {hasPurpose && (
-          <ReviewSection
-            buttonText={Config.formation.general.editButtonText}
-            header={Config.formation.fields.businessPurpose.label}
-            stepName={"Business"}
-            testId="edit-business-purpose-step"
-          >
-            <ReviewText fieldName={"businessPurpose"} isExpandable={true} />
-          </ReviewSection>
-        )}
-        <ReviewRegisteredAgent />
-        {hasMembers && !isLP && <ReviewMembers />}
-        <ReviewSignatures />
-        <ReviewBillingContact />
-        <ReviewBillingServices />
+        <ReviewMainSection
+          header={BusinessFormationStepsConfiguration[1].name}
+          stepName={"Business"}
+          testId="edit-business-name-step"
+        >
+          <BusinessNameAndLegalStructure isReviewStep />
+          <ReviewBusinessSuffixAndStartDate />
+          {isForeignCCorp && (
+            <>
+              <ReviewWillPracticeLaw willPracticeLaw={state.formationFormData.willPracticeLaw} />
+              <ReviewForeignCertificate foreignGoodStandingFile={state.foreignGoodStandingFile} />
+            </>
+          )}
+          <ReviewMainBusinessLocation />
+
+          {isLP && (
+            <ReviewSubSection
+              header={Config.formation.fields.combinedInvestment.label}
+              marginOverride="margin-top-4"
+            >
+              <ReviewText fieldName={"combinedInvestment"} />
+            </ReviewSubSection>
+          )}
+          {isLP && (
+            <ReviewSubSection
+              header={Config.formation.fields.withdrawals.label}
+              marginOverride="margin-top-4"
+            >
+              <ReviewText fieldName={"withdrawals"} />
+            </ReviewSubSection>
+          )}
+          {isLP && <ReviewPartnership />}
+          {isLP && (
+            <ReviewSubSection
+              header={Config.formation.fields.dissolution.label}
+              marginOverride="margin-top-4"
+            >
+              <ReviewText fieldName={"dissolution"} />
+            </ReviewSubSection>
+          )}
```

Comment:
Could we consolidate these `isLP` assertions?

Reply:
Consolidate? Should we create a chore for this versus addressing it in this story?

Reply:
This is what I'm proposing.
```suggestion
          {isLP && (
            <ReviewSubSection
              header={Config.formation.fields.combinedInvestment.label}
              marginOverride="margin-top-4"
            >
              <ReviewText fieldName={"combinedInvestment"} />
            </ReviewSubSection>
            <ReviewSubSection
              header={Config.formation.fields.withdrawals.label}
              marginOverride="margin-top-4"
            >
              <ReviewText fieldName={"withdrawals"} />
            </ReviewSubSection>
          <ReviewPartnership />
            <ReviewSubSection
              header={Config.formation.fields.dissolution.label}
              marginOverride="margin-top-4"
            >
              <ReviewText fieldName={"dissolution"} />
            </ReviewSubSection>
          )}
```

Reply:
I marked it resolved, b/c I updated the code. That's how I go through the feedback to ensure I have addressed all of the reviewer's comments. You can see the updates when I push the code again.

Reply:
Makes sense. I just didn't see the change. Wanted to make sure this was addressed.